File: TugasPraktikum05/Bahasa C/PRAK503-2310817320013-AikoAnatashaWendiono.c
#include <stdio.h>

int minimal(int a, int b) {
    return (a < b) ? a : b;
}

int maksimal(int a, int b) {
    while (a > b) {
        return a;
    }
    return b;
}

int main() {
    int N;
    scanf("%d", &N);

    int maks = -100000;
    int minim = 100000;

    for (int i = 0; i < N; i++) {
        int nilai;
        scanf("%d", &nilai);

        maks = maksimal(maks, nilai);
        minim = minimal(minim, nilai);
    }

    printf("%d %d", maks, minim);

    return 0;
}

File: TugasPraktikum05/Bahasa Python/PRAK503-2310817320013-AikoAnatashaWendiono.py
def minimal(a, b):
    return a if a < b else b

def maksimal(a, b):
    return a if a > b else b

def main():
    n = int(input())
    bil = list(map(int, input().split()))

    maks = -100000
    minim = 100000

    for nilai in bil:
        maks = maksimal(maks, nilai)
        minim = minimal(minim, nilai)

    print(f"{maks} {minim}")

main()
